pkg/web/apps/apps_service.go

```diff
@@ -2,30 +2,29 @@ package apps
 
 import (
 	"github.com/aerogear/mobile-security-service/pkg/models"
-	"github.com/labstack/echo"
 )
 
 type (
 	// Service defines the interface methods to be used
 	Service interface {
-		GetApps(ctx echo.Context) (*[]models.App, error)
+		GetApps() (*[]models.App, error)
 	}
 
 	appsService struct {
-		psqlRepository PostgreSQLRepository
+		repository Repository
 	}
 )
 
 // NewService instantiates this service
-func NewService(psqlRepository PostgreSQLRepository) Service {
+func NewService(repository Repository) Service {
 	return &appsService{
-		psqlRepository: psqlRepository,
+		repository: repository,
 	}
 }
 
 // GetApps retrieves the list of apps from the repository
-func (a *appsService) GetApps(c echo.Context) (*[]models.App, error) {
-	apps, err := a.psqlRepository.GetApps(c)
+func (a *appsService) GetApps() (*[]models.App, error) {
```

Comment:
Was the context object being used in this function somewhere before this change?

Reply:
it was not being used anywhere. If we wanted too, we could strip out the Go context from the echo Context to pass down to our future database calls. After talking to Craig Brookes, this is not needed.